main.js

```diff
@@ -44,6 +44,12 @@ class UI {
     books.push(book);
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static removeBook(book) {
+    const books = UI.getBooks();
+    books.splice(books.indexOf(book), 1);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also add a feature to clear the `inputs` fields once you have submitted a form to make your project looks more professional. I know you can do that :muscle: :muscle: